cli/src/commands/parse.rs

```diff
@@ -48,7 +48,7 @@ pub fn try_get_packages(path: &Path) -> Result<(Vec<PackageDescriptor>, PackageT
         let parser = format.parser();
         if let Ok(pkgs) = parser.parse(data.as_str()) {
             if !pkgs.is_empty() {
-                print_user_success!("Identified lockfile type: {}", format);
+                print_user_warning!("Identified lockfile type: {}", format);
```

Comment:
This feels very wrong? We shouldn't use `warning` only because it prints to stderr instead of stdout. That isn't really a solution, that's just a hacky workaround.

Reply:
Maybe if the `parse` command had a `--json` flag like the other commands that can produce JSON output? Would that help...like to use the `pretty_print` predicate as is done elsewhere?

Reply:
The output in this function definitely feels a bit weird... Perhaps these should just be log messages rather than `print` or `eprint`?

Reply:
Yeah logging was my first thought too, but we **definitely** want to print which lockfile was identified and only error and warn are printed by default.

Reply:
Alright. I've updated the code to use a `pretty_print` condition and continue to use stdout.